newm-server/src/main/kotlin/io/newm/server/features/user/UserRoutes.kt

```diff
@@ -47,6 +49,11 @@ fun Routing.createUserRoutes() {
     }
 
     route(USERS_PATH) {
+        get {
+            with(call) {
+                respond(repository.getAll(offset ?: 0, limit ?: 10))
```

Comment:
Should we make the default a variable to share it across, seems like the limit is the same in multiple places

Reply:
Most likely these values will change and not be equal. But if they stay the same I will do that.